### ISSUE TITLE
Alphabetic style

### ISSUE THREAD

**Issue by michal-h21** [michal-h21/biblatex-iso690#54]
Based on user's request, I've added `iso-alphabetic` style. It is just modified normal `alphabetic` style. It seems to work, but any comments and thoughts are welcomed.

**Comment by DavidLuptak** [comment]
Well, I think there are many things to consider:
* the ISO 690 standard prescribes just three methods of citation (author-year, numeric and running notes)
  * however, it's stated just in an informative annex A
  * *running notes* method is not implemented yet at all
  * `iso-authortitle` is implemented over and above (discussed many times, e.g. in #24 [here](https://github.com/michal-h21/biblatex-iso690/issues/24#issuecomment-216350831), #19 [here](https://github.com/michal-h21/biblatex-iso690/issues/19#issuecomment-162304791),..). Do we need it at all?
* `iso-alphabetic` is very similar to `iso-numeric`, isn't it?
  * there are just different sorting scheme (`nyt` vs `nty`)
  * different bibliography environment
  * however, these differences can be easily overwritten in the typeset document without touching the `biblatex-iso690` package, can't they?
    * by adding `sorting=nyt`, `labelalpha=true` to the `biblatex` package options
    * by adding [these lines](https://github.com/michal-h21/biblatex-iso690/blob/master/iso-alphabetic.bbx#L14-L44)
    * and I suppose it will be similar case also with `.cbx` citation file

So there is a main question if we want such styles in the package or we can maybe create separate branch for the *non-iso 690* styles or cover some style customizations in some *style guide* or ..?

**Comment by michal-h21** [comment]
Yes, these styles aren't defined in the standard, but almost every institution uses some modifications, because of traditions or for some some other reasons. I am in favor for helper files for these non-standard styles, because I generally think that it is a bad idea to put too much local redefinitions in the document preamble. For example, what it interface break in some `biblatex` update and the user has to update dozens of documents? (Yes, I know that it is unlikely, but sometimes you want to compile your old documents). 

Maybe we should mark these styles as `non-standard` in the documentation, maybe move them to some  subdirectory, so  it is obvious that it is not something what users should use unless they know what they do. 

>  `iso-alphabetic` is very similar to `iso-numeric`, isn't it? 

Not really, it uses short alphabetic marks in the text, like `[Bra08]`, there are used as labels in the bibliography.

> So there is a main question if we want such styles in the package or we can maybe create separate branch for the non-iso 690 styles or cover some style customizations in some style guide or ..?

I would clearly state which styles are conforming to the standard and which not.

BTW., speaking of the style guide, we should include it in the documentation as FAQ or something, it is possible to clone the Wiki using `https://github.com/michal-h21/biblatex-iso690.wiki.git` as URL, convert the Markdown file with Pandoc and include it in the TeX file. I will try to make Makefile for that.

**Comment by michal-h21** [comment]
Something like this Makefile should do the trick to compile the style guide to TeX:

```
style-guide.tex: biblatex-iso690.wiki/Style-Guide.md biblatex-iso690.wiki
	pandoc biblatex-iso690.wiki/Style-Guide.md -o style-guide.tex

biblatex-iso690.wiki/Style-Guide.md: biblatex-iso690.wiki

biblatex-iso690.wiki:
	git clone https://github.com/michal-h21/biblatex-iso690.wiki.git
```

I can see that there is a problem with citation the style guide, which doesn't came nicely in the resulting TeX file. The problem is that Github version of Markdown doesn't support footnotes nor citations :(
